src/main/java/seedu/address/logic/commands/update/UpdateNewCommand.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.logic.commands.update;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PROPERTIES;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.property.Property;
+import seedu.address.model.property.status.Offer;
+import seedu.address.model.property.status.Option;
+import seedu.address.model.property.status.Status;
+
+public class UpdateNewCommand extends UpdateCommand {
+
+    public static final String MESSAGE_PRE_EXISTING_STATUS = "Property already has a status";
```

Comment:
If you are moving the `MESSAGE_NULL_STATUS` to `UpdateCommand` class, then perhaps this should be moved as well for consistency

Reply:
moved it!